vue2/src/store/index.js

```diff
@@ -1,10 +1,41 @@
 import { createStore } from 'vuex'
-
+import { years } from '../views/Home'
 export default createStore({
   state: {
-    // message:'lalala'
+    sex: '',
+    year: years[0].year,
+    month: '1',
+    day: '1',
+    page2_1:'',
+    page2_2:'',
+    page2_3: '',
+    consultation:'',
   },
   mutations: {
+    changeSex(state,value) {
```

Comment:
細かいのですが、,の後ろにはスペースを空けましょう。
フォーマットの自動整形を使用すると良いです。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840

```suggestion
    changeSex(state, value) {
```